Modelica/Electrical/Batteries/Examples/CCCV_StackRC.mo

```diff
@@ -2,16 +2,17 @@ within Modelica.Electrical.Batteries.Examples;
 model CCCV_StackRC
   "Charge a transient stack with constant current - constant voltage characteristic"
   extends Modelica.Icons.Example;
+  parameter Modelica.Units.SI.Current Isc = 1200 "Short-circuit current of cell at OCVmax";
   parameter Modelica.Electrical.Batteries.ParameterRecords.TransientData.ExampleData cellDataOriginal(
     Qnom=18000,
     useLinearSOCDependency=false,
-    Ri=4.2/1200,
+    Ri=cellDataOriginal.OCVmax/Isc,
     Idis=0.001) "Original cell data"
     annotation (Placement(transformation(extent={{-40,-60},{-20,-40}})));
   parameter Modelica.Electrical.Batteries.ParameterRecords.TransientData.ExampleData cellDataDegraded(
     Qnom=18000,
     useLinearSOCDependency=false,
-    Ri=2*4.2/1200,
+    Ri=cellDataDegraded.OCVmax/Isc,
```

Comment:
```suggestion
    Ri=2*cellDataDegraded.OCVmax/Isc,
```
Previously it was 2*4.2/1200, and OCVmax=4.2, Isc=1200; but the factor 2 was lost.
Without this change the I don't see any difference between original and degraded.